src/js/index.js

```diff
@@ -0,0 +1,13 @@
+export {
```

Comment:
Why do we have both this file and `./components`? Seems like it would be easy to accidentally forget updating one or the other, can we combine them?

Reply:
Yeah, it's a good point. The intention was to share the set of exports from `components/index.js` between the package entrypoint and the browser "auto" bundle, but considering that the package entrypoint is already a side-effect-free set of exports, the auto bundle could just read from it.

I'll plan to just import each component from this file instead.

Alternatively, we could use [`export * from './components';`](https://github.com/tc39/proposal-export-ns-from). I was under the wrong impression that the proposal hadn't progressed very far, but looks like it's readily available now.

To the general concern of forgetting to keep things in sync, this was the idea with the tests in `test/components/index.test.js`, since we'll have more USWDS components arriving with 2.9.0 (#159) and I'd worried I'd forget to include them 😄  though in that case it's less within our control, since those components are defined in USWDS>

Reply:
Updated in 9672158 (using `export * from` syntax).